src/index.ts

```diff
@@ -473,11 +479,16 @@ export function graphqlHTTP(options: Options): Middleware {
           undefined,
           error,
         );
-        sendPartialResponse(pretty, response, {
-          data: undefined,
-          errors: [formatErrorFn(graphqlError)],
-          hasNext: false,
-        });
+        sendPartialResponse(
+          pretty,
+          response,
+          {
+            data: undefined,
+            errors: [formatErrorFn(graphqlError)],
+            hasNext: false,
+          },
+          false,
+        );
       }
       response.write('\r\n-----\r\n');
```

Comment:
need to remove this line to prevent sending double terminating boundary?

Reply:
The "sendPartialResponse" wont include the boundary if `hasNext` is false, so the boundary will need to be sent here. Alternatively in the `sendPartialResponse` method in the else case, we could probably include there instead. Your call!

Reply:
i think it's better to include it in `sendPartialResponse`, otherwise there's a delay while we wait for the asyncIterable to close out (which is another async operation)

Reply:
makes sense! 🎉